example/package.json

```diff
@@ -14,9 +14,9 @@
     "lint": "prettier -c --cache . && eslint . --cache --max-warnings=0"
   },
   "dependencies": {
-    "@astrojs/starlight": "0.11.0",
-    "astro": "3.2.3",
```

Comment:
I see that you like to pin the versions @HiDeoo. However I think that is usually a mistake. Would you mind explaining your reasoning? I assume you know that the lockfile will prevent "works on my machine" without pinning versions in the `package.json`.

Reply:
FWIW: I have read what [renovate](https://docs.renovatebot.com/dependency-pinning/) says about pinning. I think they are wrong, ["surprise updates breaking the code"](https://docs.renovatebot.com/dependency-pinning/#what-a-lock-file-doesnt-do-for-you) have rarely been an issue for me and I think the [sub-dependency argument](https://docs.renovatebot.com/dependency-pinning/#what-about-indirectsub-dependencies) renders all arguments for pinning mute.

Nevertheless, even if one follows their advise, the dependencies in the `packages/starlight-blog/package.json` should not be pinned. At least that is what they say.

Reply:
Let me know if in general you like the PR. If so I will repin the versions, if you want me to.